configs/helm/README.md

```diff
@@ -0,0 +1,66 @@
+| alertmanager.service.port        | int    | `9093`                                |                                                    |
+| grafana.container.image          | string | `"grafana/grafana:7.3.6"`             | Docker image for grafana                           |
+| grafana.container.port           | int    | `3000`                                |                                                    |
+| grafana.enabled                  | bool   | `true`                                |                                                    |
+| grafana.ingress.class            | string | `"nginx"`                             | Ingress class to use for grafana                   |
+| grafana.ingress.hostname         | string | `"monitor.tellor.io"`                 | Hostname to use for accessing grafana              |
+| grafana.ingress.path             | string | `"/"`                                 | Subpath to access grafana                          |
+| grafana.ingress.tls.enabled      | bool   | `false`                               | Enable/Disable TLS for grafana                     |
+| grafana.ingress.tls.secret       | string | `"grafana-tls-secret"`                | Name of TLS secret to use for grafana              |
+| grafana.persist                  | bool   | `true`                                | Enable persistance for grafana configuration       |
+| grafana.service.port             | int    | `80`                                  |                                                    |
+| grafana.storage                  | string | `"5Gi"`                               | Grafana persistent storage size                    |
+| prometheus.container.image       | string | `"prom/prometheus:v2.24.0"`           | Docker image for prometheus                        |
+| prometheus.container.port        | int    | `9090`                                |                                                    |
+| prometheus.enabled               | bool   | `true`                                |                                                    |
+| prometheus.persist               | bool   | `true`                                | Enable persistance for prometheus configuration    |
+| prometheus.service.port          | int    | `9090`                                |                                                    |
+| prometheus.storage               | int    | `50`                                  | Prometheus storage size in GB                      |
+| telliot.container.image          | string | `"tellor/telliot:latest"`             | Docker image for telliot                           |
+| telliot.container.port           | int    | `9090`                                |                                                    |
+| telliot.modes                    | string | `"{dataServer,mine}"`                 | Array of arguments to spawn telliot instances with |
```

Comment:
maybe just call it `command` and show an example on how to run it in mine and dataserver